app/components/channel_modal.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {Component, PropTypes} from 'react';
+import {Animated, Dimensions} from 'react-native';
+
+const AnimatedView = Animated.View;
+const {width: deviceWidth, height: deviceHeight} = Dimensions.get('window');
+
+export default class ChannelModel extends Component {
+    static propTypes = {
+        children: PropTypes.node,
+        duration: PropTypes.number,
+        topOffset: PropTypes.number,
+        theme: React.PropTypes.object.isRequired,
```

Comment:
@jarredwitt This line looks like a copy pasta crumb from another file; the prop is both required & unused and we're unnecessarily accessing  `PropTypes` through `React`, which is being imported.